model_info/kowiki.reverted.md

```diff
@@ -15,67 +15,67 @@ Model Information:
 	 - python_implementation: 'CPython'
 	 - python_revision: ''
 	 - python_version: '3.5.3'
-	 - release: '4.9.0-6-amd64'
+	 - release: '4.9.0-8-amd64'
 	
 	Statistics:
-	counts (n=19779):
+	counts (n=19742):
 		label        n         ~True    ~False
 		-------  -----  ---  -------  --------
-		True       591  -->      162       429
-		False    19188  -->      303     18885
+		True       951  -->      414       537
+		False    18791  -->     1137     17654
 	rates:
 		              True    False
 		----------  ------  -------
-		sample       0.03     0.97
+		sample       0.048    0.952
 		population   0.047    0.953
-	match_rate (micro=0.927, macro=0.5):
+	match_rate (micro=0.882, macro=0.5):
 		  False    True
 		-------  ------
-		  0.972   0.028
-	filter_rate (micro=0.073, macro=0.5):
+		  0.922   0.078
+	filter_rate (micro=0.118, macro=0.5):
 		  False    True
 		-------  ------
-		  0.028   0.972
-	recall (micro=0.951, macro=0.629):
+		  0.078   0.922
+	recall (micro=0.916, macro=0.687):
 		  False    True
 		-------  ------
-		  0.984   0.274
-	!recall (micro=0.308, macro=0.629):
+		  0.939   0.435
+	!recall (micro=0.459, macro=0.687):
 		  False    True
 		-------  ------
-		  0.274   0.984
-	precision (micro=0.941, macro=0.713):
+		  0.435   0.939
+	precision (micro=0.938, macro=0.617):
 		  False    True
 		-------  ------
-		  0.965   0.462
-	!precision (micro=0.486, macro=0.713):
+		  0.971   0.263
+	!precision (micro=0.296, macro=0.617):
 		  False    True
 		-------  ------
-		  0.462   0.965
-	f1 (micro=0.945, macro=0.659):
+		  0.263   0.971
+	f1 (micro=0.925, macro=0.641):
 		  False    True
 		-------  ------
-		  0.974   0.344
-	!f1 (micro=0.374, macro=0.659):
+		  0.955   0.328
+	!f1 (micro=0.357, macro=0.641):
 		  False    True
 		-------  ------
-		  0.344   0.974
-	accuracy (micro=0.951, macro=0.951):
+		  0.328   0.955
+	accuracy (micro=0.916, macro=0.916):
 		  False    True
 		-------  ------
-		  0.951   0.951
-	fpr (micro=0.692, macro=0.371):
+		  0.916   0.916
+	fpr (micro=0.541, macro=0.313):
 		  False    True
 		-------  ------
-		  0.726   0.016
-	roc_auc (micro=0.871, macro=0.87):
+		  0.565   0.061
+	roc_auc (micro=0.856, macro=0.856):
```

Comment:
Seems unhealthier, why?

Reply:
It's a pretty minor change.  I bet it had something to do with changes in our autolabeler.